3.3/test/setup-test-app/testapp/config.py

```diff
@@ -1 +0,0 @@
-# you can put gunicorn configuration options here, see http://docs.gunicorn.org/en/latest/configure.html
```

Comment:
Why removing that comment from here, it was placed purposely there to inform users about possible contents of this file. Having it empty doesn't make sense.